device-registry/src/main/scala/org/genivi/sota/device_registry/DevicesResource.scala

```diff
@@ -133,6 +133,11 @@ class DevicesResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
         .map(ActiveDeviceCount(_)))
     }
 
+  def getDistinctPackages(ns: Namespace): Route =
+    parameters('offset.as[Long].?, 'limit.as[Long].?) { (offset, limit) =>
+      complete(db.run(InstalledPackages.getInstalledForAllDevices(ns, offset, limit)))
```

Comment:
The two endpoints which do pagination so far just return Seq[T] as they are used by core and changing their return value requires refactoring. However, since this endpoint isn't used yet, I think it would be good to adopt the practice of returning the offset, limit and total number of records/values, as well as the values themselves. The idea being that then the client does not need to keep track of which page it is on.